### ISSUE TITLE
Notebook failed on VS Code with Jupyter extension

### ISSUE THREAD

**Issue by trgiangdo** [Avaiga/taipy-getting-started#30]
* Taipy version: 1.1.0
* Python version: 3.10.6
* Operating System: Windows

### Description

I tried to run the Getting started Notebook on VS Code with Jupyter extension and get this error.
```
ipykernel_launcher.py: error: argument --force-https: ignored explicit argument 'c:\\Users\\Acer\\AppData\\Roaming\\jupyter\\runtime\\kernel-v2-14083YMFPZiWPH8D.json'
Output exceeds the size limit. Open the full output data in a text editor
---------------------------------------------------------------------------
ArgumentError                             Traceback (most recent call last)
File c:\Users\Acer\AppData\Local\Programs\Python\Python310\lib\argparse.py:1859, in ArgumentParser.parse_known_args(self, args, namespace)
   1858 try:
-> 1859     namespace, args = self._parse_known_args(args, namespace)
   1860 except ArgumentError:

File c:\Users\Acer\AppData\Local\Programs\Python\Python310\lib\argparse.py:2068, in ArgumentParser._parse_known_args(self, arg_strings, namespace)
   2067     # consume the next optional and any arguments for it
-> 2068     start_index = consume_optional(start_index)
   2070 # consume any positionals following the last Optional

File c:\Users\Acer\AppData\Local\Programs\Python\Python310\lib\argparse.py:1990, in ArgumentParser._parse_known_args.<locals>.consume_optional(start_index)
   1989         msg = _('ignored explicit argument %r')
-> 1990         raise ArgumentError(action, msg % explicit_arg)
   1992 # if there is no explicit argument, try to match the
   1993 # optional's string arguments with the following strings
   1994 # if successful, exit the loop
   1995 else:

ArgumentError: argument --force-https: ignored explicit argument 'c:\\Users\\Acer\\AppData\\Roaming\\jupyter\\runtime\\kernel-v2-14083YMFPZiWPH8D.json'

During handling of the above exception, another exception occurred:

SystemExit                                Traceback (most recent call last)
...
    170 if isinstance(error, str):
    171     error = AssertionError(error)
--> 172 raise error

AssertionError: 
```

### What I Did

1. Install `ipykernel` and `taipy`.
1. On Visual Studio Code, install [**Jupyter** extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter).
1. Open `getting_started.ipynb` inside VS Code and run the code cells.

**Comment by trgiangdo** [comment]
This bug was fixed in [taipy-gui/#377](https://github.com/Avaiga/taipy-gui/pull/377) on `develop` branch.

It's seen that the fix was not in taipy_gui-1.1.3 yet.

Moreover, the bug is only on Jupyter extension on VS Code so it's not that important.

**Comment by jrobinAV** [comment]
Do we plan to fix the 1.1.3 branch?
